lists.py

```diff
@@ -33,4 +33,7 @@ def myfunc(*args):
 print(eletters)
 
 for l in eletters:
-    print(l)
\ No newline at end of file
+    print(l)
+
```

Comment:
Nice use of f-string! Consider adding a docstring so others know what this function does